src/CattyTests/Resources/ParserTests/Custom/ValidProjectAllBricks/ValidProjectAllBricks0992.xml

```diff
@@ -741,6 +741,15 @@
                     <commentedOut>false</commentedOut>
                     <instrumentSelection>PIANO</instrumentSelection>
                 </brick>
+                <brick type="SetBackgroundByIndexBrick">
+                  <commentedOut>false</commentedOut>
+                  <formulaList>
+                    <formula category="SetBackgroundByIndexBrick">
```

Comment:
This is an invalid XML. There should be two bricks in it with the category named (a) `LOOK_INDEX` and (b) `BACKGROUND_INDEX`.